R/prep_calculate_portfolio_aggregate_scores.R

```diff
@@ -0,0 +1,27 @@
+prep_calculate_portfolio_aggregate_scores <- function(data,
```

Comment:
nit: potentially we could drop the `prep_` prefix in all functions except for `prep_scores()`. To me the `prep_*()` functions are directly linked to the `plot_*()` functions.

Reply:
done